index.html

```diff
@@ -0,0 +1,14 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="title">Hello Microverse!</h1>
```

Comment:
Your HTML file currently lacks a closing `</body>` tag. 
I think you should add the tag to your file. 😃

Reply:
Thank you